src/styles_css/product_card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: var(--card-width);
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__product-image {
+  width: var(--card-product-img-width);
+  height: var(--card-product-img-height);
+  margin-bottom: 40px;
+}
+
+.card__product-title {
+  font-weight: 500;
+  font-size: var(--card-medium-font-size);
+  line-height: 18px;
+  color: var(--card-color-dark);
+
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-size: var(--card-small-font-size);
+  line-height: 14px;
+  color: var(--card-color-grey);
+
+  align-self: start;
+  margin-bottom: 16px;
+}
+
+.card__elements-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: var(--card-elements-wrapper-width);
+}
+
+.card__product-reviews {
+  font-size: var(--card-small-font-size);
+  line-height: 14px;
+  color: var(--card-color-dark);
+}
+
+.card__elements-wrapper--reviews {
+  margin-bottom: 24px;
+}
+
+.card__product-price-title {
+  font-size: var(--card-medium-font-size);
+  line-height: 18px;
+
+  color: var(--card-color-grey);
+}
+
+.card__product-price {
+  font-weight: 700;
+  font-size: var(--card-extra-large-font-size);
+  line-height: 18px;
+
+  color: var(--card-color-dark);
+}
+
+.card__elements-wrapper--price {
+  margin-bottom: 16px;
+}
+
+.card__buy-link {
+  display: flex;
+  justify-content: center;
+  font-weight: 700;
+  font-size: var(--card-large-font-size);
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: var(--card-color-white);
+
+  width: 100%;
```

Comment:
flex element will take up all of the available width anyways
```suggestion
```

Reply:
when i remove width, my button squash to text width and dont take all available width